test/trento/application/usecases/sap_systems_test.exs

```diff
@@ -48,12 +48,14 @@ defmodule Trento.SapSystemsTest do
              ] = SapSystems.get_all_sap_systems()
     end
 
-    test "should retrieve all the existing databases and the related instances" do
+    test "should retrieve all the currently registered existing databases and the related instances" do
       %DatabaseReadModel{
         id: sap_system_id,
         sid: sid
       } = insert(:database)
 
+      insert(:database, deregistered_at: DateTime.utc_now())
```

Comment:
Shouldn't this cause a failure in the test?

Reply:
At the very least, shouldn't we verify that this database is NOT returned from `SapSystems.get_all_databases()`